### PR TITLE
Section 3

### PR DESCRIPTION
**Pull request by chicco785**

### CODE REVIEW

**Comment by chicco785** [comment]
@martel-innovate/fiware-lab link to the build: https://join-fiware-lab-guide.readthedocs.io/en/section-3/

File: docs/3.process/1.services.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+are required to install the following OpenStack services based on the
+'''OpenStack Newton release''':
+
+-   Mandatory:
+
+    -   OpenStack Nova (using KVM as hypervisor since image catalogue
+        stores KVM compatible images).
+
+    -   OpenStack Glance (Swift as default backend type, the solution
+        may be adopted depending on hardware owned by the specific
```
Comment (taliaga) [inline docs/3.process/1.services.md:16]: adopted or adapted?

Comment (taliaga) [reply docs/3.process/1.services.md:16]: or maybe you meant as below "**other** solution**s** may be adpoted"

File: docs/3.process/1.services.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        FIWARE Lab node).
+
+    -   OpenStack Cinder (as default solution we suggest LVM, other
+        solution may be adopted depending on hardware owned by the
+        specific FIWARE Lab node).
+
+    -   OpenStack Neutron with OVS and GRE or VxLAN tunnels (floating
+        IPs must be made available to users. The minimum number is
+        currently under discussion).
```
Comment (taliaga) [inline docs/3.process/1.services.md]: This is inconsistent with section 1. Still in discussion or 20 + one per OS service?

File: docs/3.process/1.services.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    -   OpenStack Ceilometer with MongoDB as backed as default solution.
+
+    -   OpenStack Keystone only for initial setup and testing, then
+        FIWARE Lab keystone should be used[^1].
```
Comment (taliaga) [inline docs/3.process/1.services.md]: where's this ref 1?

File: docs/3.process/1.services.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    -   OpenStack Horizon only for initial setup and testing, then
+        FIWARE Lab Cloud Portal should be used.
+
+-   Optional
+
+    -   OpenStack Swift with 3 replication factor value. Optionally,
+        could be installed CEPH with the OpenStack Swift APIs.
```
Comment (taliaga) [inline docs/3.process/1.services.md]: ... CEPH could be installed ...

File: docs/3.process/2.upgrade.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+FIWARE Lab nodes are based on OpenStack. OpenStack is developed and
+released around 6-month cycle.
+
+According to FIWARE Lab management rules the upgrade policy of a FIWARE
+Lab node is two version behind the current official version under
+development. This is to avoid unsupported-EOL OpenStack release,
+security and performance issues. This policy secures us that we are
+almost in line with the community. It is important to notice that the
+upgrade of the OpenStack version can involve also the upgrade of the
+Operative System. The recommendation of the FIWARE Lab is the use of
```
Comment (taliaga) [inline docs/3.process/2.upgrade.md]: Operating System

File: docs/3.process/2.upgrade.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+Ubuntu like Operating System. The next image shows as detail about the
```
Comment (taliaga) [inline docs/3.process/2.upgrade.md:13]: ...shows in detail the FIWARE...
or 
...shows [more] details about the...

File: docs/3.process/2.upgrade.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+FIWARE Lab policy in use:
+
+![FIWARE Lab OpenStack support model](image3.png)
+
+More information about the release series of OpenStack can be found at
+the following link: [https://releases.openstack.org](https://releases.openstack.org)
+
+* **IMPORTANT**: A FIWARE Lab Node not updated, will stop working properly because
+  not compatible with FIWARE Lab services!
```
Comment (taliaga) [inline docs/3.process/2.upgrade.md:22]: - remove comma above.
- ... because of incompatibilities...

File: docs/3.process/3.installing.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
+## Installing FIWARE Lab Node 
+
+### Introduction
+
+In order to install your FIWARE Lab Node you can choose among different
+options that allows you to deploy an updated version of Vanilla
+OpenStack compatible with the requirements listed in section [Essential things](../1.essentials).
```
Comment (taliaga) [inline docs/3.process/3.installing.md:7]: is this link correct?

File: docs/3.process/3.installing.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+### <a name="how"></a> How to install a FIWARE Lab Node
+
+Currently, the OpenStack community offers many ways to install a
+complete environment and the procedure can be manual or automatic.
+Thanks to novel DevOps techniques the current trend is to leverage
+Infrastructure as Code concept and IT Automation tools like Ansible,
+Puppet or Chef in order to provision and maintain such complex systems.
+
+Moreover Operating System Virtualization (Container-based) helps the
```
Comment (taliaga) [inline docs/3.process/3.installing.md:17]: Moreover,

File: docs/3.process/3.installing.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+    1. [*https://docs.openstack.org/install-guide/*](https://docs.openstack.org/install-guide/)
+
+    1. [*https://docs.openstack.org/install/*](https://docs.openstack.org/pike/install/)
+
+### Suggested deployment architecture
+
+To join FIWARE Lab no minimal requirement is enforced but the
+infrastructure must be adequate to support the needs of users who will
+be hosted on the new nodes. Obviously during the first node setup may
```
Comment (taliaga) [inline docs/3.process/3.installing.md:60]: ... it may not...

File: docs/3.process/4.configuring.md [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+The default flavors should be:
+
+| **ID** | **Name** | **Memory (MB)** | **Disk (Gb)** | **Ephemeral** | **Swap** | **vCPUs** | **RXTX Factor** | **Public** |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | 
+| 1 | m1.tiny | 512 | 1 | 0 | | 1 | 1.0 | True |
+| 2 | m1.small | 2048 | 20 | 0 | | 1 | 1.0 | True |
+| 3 | m1.medium| 4096 | 40 | 0 | | 2 | 1.0 | True |
+| 4 | m1.large | 8192 | 80 | 0 | | 4 | 1.0 | True |
+
+For the nova service, default quotas the values that should be defined
```
Comment (taliaga) [inline docs/3.process/4.configuring.md]: the default quota values that ...

File: docs/3.process/5.statuscheck.md [hunk 12]
```diff
@@ -0,0 +1,283 @@
...
+    We should verify that the neutron server and the different neutron
+    agents (metadata, dhcp, l3 and openvswitch) and running properly.
+   
+    ```
+    $ service neutron-server status
+   
+    neutron-server start/running, process 63925
+   
+    $ for srv in metadata dhcp l3 openvswitch; do service
+    neutron-srv-agent status; done
```
Comment (taliaga) [inline docs/3.process/5.statuscheck.md]: missing $

File: docs/3.process/5.statuscheck.md [hunk 13]
```diff
@@ -0,0 +1,283 @@
...
+| d92c1377-c51a-4307-8a75-48df8d251e5d | Open vSwitch agent | node-29 | :-) | True | neutron-openvswitch-agent |
+| e4e0aa0b-2698-41c4-a36e-48580aba640f | Open vSwitch agent | node-28 | :-) | True | neutron-openvswitch-agent |
+| ec05e2e0-bcfe-48cc-ace5-8c612f164604 | Metadata agent | node-30 | :-) | True  | neutron-metadata-agent |
+
+### OpenStack Cinder
+
+There are 4 services that have to be checked, depending on the
+configuration of your OpenStack instance. Those services are cinder-api,
+cinder-scheduler, cinder-volume and cinder-backup (this one sometimes is
+not installed in the configuration. Therefore, we have to test.
```
Comment (taliaga) [inline docs/3.process/5.statuscheck.md]: closing ) missing

File: docs/3.process/7.connecting.md [hunk 14]
```diff
@@ -0,0 +1,223 @@
...
+Node to FIWARE Lab.
+
+### Prerequisites
+
+Before starting the process of connecting your node to FIWARE Lab you
+have to accomplish the following:
+
+-   To be an authorized node to join FIWARE Lab, you can get this
+    authorization by filling the corresponding document
+    [Become a new FIWARE Lab node](../D) and send it to FIWARE Lab administrator
```
Comment (taliaga) [inline docs/3.process/7.connecting.md]: TODO: link

File: docs/3.process/7.connecting.md [hunk 15]
```diff
@@ -0,0 +1,223 @@
...
+
+### Node publication in FIWARE Lab
+
+Once the registration process is finished and you have correctly
+validated it using the CLI tools, your node is ready for publication in
+FIWARE Lab.
+
+To publish it you have to contact the FIWARE Lab Keystone administrators
+telling them you have finished the process. They will check everything
+is ok and the will activate your node in the FIWARE Lab Cloud Portal and
```
Comment (taliaga) [inline docs/3.process/7.connecting.md]: typo: they

Comment (taliaga) [reply docs/3.process/7.connecting.md]: May be useful to have in this section the exact pieces of information the FIWARE Lab admins will need from the new Node, if known at this time of writing of course :)

**Review by taliaga — APPROVED** [review]

**Review by acca — APPROVED** [review]

**Pull request merged**